drupal/composer.lock

```diff
@@ -2609,9 +2609,6 @@
                         "[web-root]/profiles/README.txt": "assets/scaffold/files/profiles.README.txt",
                         "[web-root]/themes/README.txt": "assets/scaffold/files/themes.README.txt"
                     }
-                },
```

Comment:
@petrillek don't we need this patch anymore? It's not in composer.json and therefore is automatically removed from composer.lock